### PR TITLE
CAPVCD: Release v29.1.0.

### PR DESCRIPTION
**Pull request by Gacko**
### Checklist

- [x] Roadmap issue created
- [x] Release uses latest stable Flatcar
- [x] Release uses latest Kubernetes patch version

### Triggering E2E tests

To trigger the E2E test for each new Release added in this PR, add a comment with the following:

`/run releases-test-suites`

If you want to trigger conformance tests, you can do so by adding a comment similar to the following:

`/run conformance-tests PROVIDER=capa RELEASE_VERSION=29.1.0`

For more details see the [README.md](/README.md#running-tests-against-prs).

### CODE REVIEW

**Comment by Gacko** [comment]
/run releases-test-suites PREVIOUS_RELEASE=29.0.0 TARGET_RELEASES=cloud-director-29.1.0

**Comment by Gacko** [comment]
@giantswarm/team-rocket This is for aligning the available features and releases to the latest releases of the other providers, so CAPA v29.4.0, CAPZ v29.3.0 and CAPV v29.1.0.

File: cloud-director/v29.1.0/release.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  - name: vertical-pod-autoscaler-crd
+    version: 3.1.1
+  components:
+  - name: cluster-cloud-director
+    catalog: cluster
+    version: 0.63.1
+  - name: flatcar
+    version: 3975.2.2
+  - name: kubernetes
+    version: 1.29.10
```
Comment (AverageMarcus) [inline cloud-director/v29.1.0/release.yaml:100]: Do we want to bump this to the latest `1.29.11`?

Comment (Gacko) [reply cloud-director/v29.1.0/release.yaml:100]: Depends if we prefer having releases aligned as much as possible over being on the latest Kubernetes. Also this release would not roll nodes until now, so could easily be updated to.

Should we maybe rather go for the latest Flatcar / Kubernetes / OS Tooling in yet another minor release across all providers at the same time?

Comment (AverageMarcus) [reply cloud-director/v29.1.0/release.yaml:100]: I'm happy with consistency with other releases and then update all releases in-line with each other

**Comment by Gacko** [comment]
/run releases-test-suites PREVIOUS_RELEASE=29.0.0 TARGET_RELEASES=cloud-director-29.1.0

**Review by njuettner — APPROVED** [review]

**Comment by Gacko** [comment]
/run releases-test-suites PREVIOUS_RELEASE=29.0.0 TARGET_RELEASES=cloud-director-29.1.0

**Pull request merged**